tests/TestCase.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Styde\LaravelQueryFilter\Tests;
```

Comment:
I think this could be simplified as QueryFilter only, as the Styde community works mostly with Laravel anyway.